Describe the Chart - Axis Label Templates demo (#259)
DevExpress/devextreme-demos: JSDemos/Demos/Charts/AxisLabelsTemplates/description.md

> Axis labels are visual elements that display values of the [major axis ticks](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/tick/).

Reviewer: I don't think you need the expression "visual elements" here. 

```suggestion
Axis labels display values for [major axis ticks](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/tick/).
```

> Axes [labels](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/) are visual elements that display values of the [major axis ticks](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/tick/).

Reviewer 2: > Axes [labels](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/)

- It should be "ax**i**s".
- We give links to labels further. Let's remove this link.

> To configure labels for individual axes, specify label settings in the **valueAxis**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/label/) and **argumentAxis**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/label/) objects. To configure labels for all axes, use the **commonAxisSettings**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/) object. Individual settings take precedence over these settings.

Reviewer: ```suggestion
To configure labels for individual axes, specify label settings in the **valueAxis**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/label/) and **argumentAxis**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/label/) objects. To configure labels for all axes, use the **commonAxisSettings**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/) object. Individual settings take precedence over common settings.
```

> To configure labels for individual axes, specify labels settings in the **valueAxis**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/label/) and **argumentAxis**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/label/) objects. To configure labels for all axes, use the **commonAxisSettings**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/) object. Individual settings take precedence over these settings.

Reviewer 2: ```suggestion
To configure labels for individual axes, specify label settings in the **valueAxis**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/label/) and **argumentAxis**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/label/) objects. To configure labels for all axes, use the **commonAxisSettings**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/) object. Individual settings take precedence over these settings.
```

> To configure labels, specify their settings in the **label** object of the [value](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/label/) or [argument](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/label/) axis. To specify options for all axes, use the **commonAxisSettings**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/) object. Individual settings take precedence over these settings.

Reviewer 2: > To configure labels, ... To specify options for all axes, ...

It is better with parallelism: "To configure labels for individual axes, ... " and "To configure labels for all axes, ...".

> the **label** object of the [value](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/label/) or [argument](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/label/) axis

We use the term "object" with label but don't do this with the axes for some reason. How about "specify label settings in the [valueAxis].[label] and [argumentAxis].[label] objects"?

> This demo shows how to display custom content in axis labels. To do this, declare SVG markup in the [template](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/#template) option. You can access a label's original and formatted values from template code. 

Reviewer 3: ```suggestion
This demo shows how to display custom content in axis labels. To do this, declare the SVG markup in the [template](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/#template) option. You can access a label's original and formatted values from template code. 
```

> This demo shows how to implement labels with custom content. To do this, declare the SVG markup in the [template](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/#template) option. You can access a label's raw and formatted values in template data. 

Reviewer: We're not really implementing new labels, we are just showing something different in them. I thought it would be more straightforward to say it like this: 

```suggestion
This demo shows how to display custom content in axis labels. To do this, declare SVG markup in the [template](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/#template) option. You can access a label's original and formatted values from template code. 
```

> This demo shows how to implement labels with custom content. To do this, declare the SVG markup in the [template](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/#template) option. You can access the label's raw or converted to string data inside the template via its parameters. 

Reviewer 2: > converted to string data

Can we use "formatted value" instead?

> inside the template via its parameters

"Parameters" can be only in functions, which is all about jQuery. How about we rewrite this sentence as follows:

"You can access a label's raw and formatted values in template data."
